operator/v1/types_network.go

```diff
@@ -96,12 +96,32 @@ type DefaultNetworkDefinition struct {
 	OVNKubernetesConfig *OVNKubernetesConfig `json:"ovnKubernetesConfig,omitempty"`
 }
 
-// AdditionalNetworkDefinition configures an extra network that is available but not
+// MacVlanConfig contains configurations for macvlan interface.
+type MacVlanConfig struct {
+
+	// Host interface to enslave interface.
+	Master string `json:"master"`
```

Comment:
"Host interface to create the macvlan interface from" ?

(Also, I feel like this is more "parent" than "master"...)

Also, for consistency with other comments, this should start with the field name in lowercase

Reply:
I take it from macvlan CNI plugin's [README.md](https://github.com/containernetworking/plugins/tree/master/plugins/main/macvlan), but agree with you and changed.